### PR TITLE
Added badges to README

### LINKED ISSUE #50: Add status badges to README.md
Might be a cool way to broadcast bot status.

### PR DESCRIPTION
**Pull request by matt-ross16**
Closes #50 

Implemented a few badges to reflect to status of enarx/bot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
+![hourly](https://github.com/enarx/bot/workflows/hourly/badge.svg)
+![cleanup-sprint-board](https://github.com/enarx/bot/workflows/cleanup-sprint-board/badge.svg)
```
Comment (mbestavros) [inline README.md:2]: - The `hourly` workflow will be removed soon-ish. While the badge makes sense now, it likely won't in a few weeks. I'm still fine with the change, though.

- The `cleanup-sprint-board` workflow will remain, though it's only ever run manually at the end of every sprint, so it's not super helpful to have it. I'd personally drop this one.

Comment (matt-ross16) [reply README.md:2]: Will there be a new Workflow added, or is it best to just move away from these badges?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+[![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/enarx/bot.svg)](http://isitmaintained.com/project/enarx/bot "Average time to resolve an issue")
+[![Percentage of issues still open](http://isitmaintained.com/badge/open/enarx/bot.svg)](http://isitmaintained.com/project/enarx/bot "Percentage of issues still open")
```
Comment (mbestavros) [inline README.md:4]: I'm not opposed to merging these, but it's worth noting that we haven't really been using issues in a high-throughput way like some of Enarx's other repos. They've mostly been used to denote larger tasks that get done in a few weeks. As such, the numbers on these badges look way worse than I believe they are.

**Review by mbestavros — APPROVED** [review]
I've left comments inline. Not opposed to merging this, but I think it has limited utility. As discussed in https://github.com/enarx/bot/issues/50, we really need these badges on repos that use the Enarxbot instead of on Enarxbot directly.

Another alternative would be to go to the other extreme and put all badges for every workflow across the Enarx org in one place. That would allow us to see if any of them are failing without having to triage them individually.

**Comment by mbestavros** [comment]
Closing this, as it's not quite what we want for #50.

**Pull request closed without merging**